src/common/BaseAnnotator.ts

```diff
@@ -60,19 +61,24 @@ export default class BaseAnnotator extends EventEmitter {
 
     store: store.AppStore;
 
-    constructor({ apiHost, container, features = {}, file, fileOptions, initialMode, intl, token }: Options) {
+    constructor({ apiHost, container, features = {}, file, fileOptions, initialMode, intl, stroke, token }: Options) {
         super();
 
         const fileOptionsValue = fileOptions?.[file.id];
         const currentFileVersionId = fileOptionsValue?.currentFileVersionId;
         const fileOptionsVersionId = fileOptionsValue?.fileVersionId;
         const fileVersionId = file.file_version.id; // This is always the currently-previewed version id, not current version id
+        const { color: defaultColor, size: defaultSize } = store.defaultStroke;
+        const { color = defaultColor, size = defaultSize } = stroke ?? {};
```

Comment:
We should defer to the reducer to set the default state rather than re-implementing it here.